python_script/meter-health/meter_health.py

```diff
@@ -0,0 +1,119 @@
+import requests
+import csv
+import logging
+logging.basicConfig(filename='example.log',level=logging.DEBUG)
+from oauthlib.oauth2 import LegacyApplicationClient
+from requests_oauthlib import OAuth2Session
+from config.credentials import *
+
+
+class MeterHealth:
+    # URL information for the API. ONLY change this method if you know what you are doing
+    BASE_URL="http://54.90.173.204:8080"
```

Comment:
This constant should be moved to a python module so the end user can modify it freely.